### PR TITLE
Remove routing code to `save_event_highcpu`

### PR DESCRIPTION
**Pull request by Swatinem**
The reason the `highcpu` variant was introduced in the first place was primarily motivated by slowness of the grouping code.
That is not the case anymore.
Events have also been routed back to the normal `save_event` queue/worker via https://github.com/getsentry/sentry-options-automator/pull/1034, and the infra seems to be happy with that.

This is the next step in removing this queue / worker pool, thus simplifying the infra and saving a couple of **$$$**.

### CODE REVIEW

File: src/sentry/tasks/store.py [hunk 1]
```diff
@@ -965,6 +960,7 @@ def save_event_attachments(
     )
 
 
+# TODO(swatinem): remove this (and related queue) once the backing worker deployment is gone
```
Comment (anonrig) [inline src/sentry/tasks/store.py:963]: Can you elaborate on this?

**Review by anonrig — COMMENTED** [review]
Can you add a description to the pull request? Do we NOT use this?

File: src/sentry/tasks/store.py (hunk 1 above)
Comment (Swatinem) [reply src/sentry/tasks/store.py:963]: I hope the description explains things, and that we are slowly removing this thing one part after the other.

**Review by anonrig — APPROVED** [review]
Thank you for the followup!

**Pull request merged**